projects/js-packages/social-logos/README.md

```diff
@@ -0,0 +1,88 @@
+- Pinterest: https://business.pinterest.com/en/brand-guidelines
+- Squarespace: http://www.squarespace.com/brand-guidelines/
+- reddit: https://www.reddit.com/about/alien/
+- Mastodon: https://joinmastodon.org/branding
+- Fediverse: https://commons.wikimedia.org/wiki/File:Fediverse_logo_proposal.svg
+- Nextdoor: https://about.nextdoor.com/gb/media/
+- http://findguidelin.es
+- Threads: https://en.wikipedia.org/wiki/File:Threads_(app)_logo.svg
+- X: https://about.twitter.com/en/who-we-are/brand-toolkit
+
+## Using the SocialLogo Component in your project:
+
+Note that this component requires [react](https://www.npmjs.com/package/react) to be installed in your project.
+
+SocialLogo renders a single social-logo svg based on an `icon` prop. It takes a size property but defaults to 24px. For greater sharpness, the icons should only be shown at either 18px, 24px, 36px or 48px. 
+
+There's a gallery with all the available icons in https://wpcalypso.wordpress.com/devdocs/design/social-logo.
+
+```
+npm install social-logos --save
+```
+#### Usage
+
+```
+import SocialLogo from 'social-logos';
+//...
+render() {
+    return <SocialLogo icon="twitter" size={ 48 } />;
+}
+```
+
+#### Props
+
+* `icon`: String - the icon name.
+* `size`: Number - (default: 24) set the size of the icon.
+* `onClick`: Function - (optional) if you need a click callback.
+
+## Notes & Pixel Grid
+
+The icon grid is based on [Gridicons](https://github.com/Automattic/gridicons) and adheres to the same rules. That is to say, the set is designed on a 24px base grid. That means logos will look their sharpest and crispest when SVGs are inserted with 24px width/height, or the icon font is used at `font-size: 24px;`. 
+
+Logos will also scale well to other sizes, like 18px (75% size), or 36px (150% size). Normally, using icon-sets outside of their pixelgrid is a surefire way to get fuzzy icons. This is also true in the case of this logo set, however unlike custom-designed icons, this is almost unavoidable in the case of logos. The problem is, every single logo is designed with its own dimensions. If we are to respect branding guidelines (which we should), no hinting or pixel-tuning is applied to any logo added to this set. Which means even at the base 24px size, logos could appear fuzzy and less than optimal. That is the way of the world, and a tradeoff between flexibility and respecting the original logo design on one hand, and pixel-perfect logos on the other hand. 
+
+So to summarize:
+
+- **Do** use Social Logos at 48px, 36px, 24px, 18px, 12px. Prioritize 24px or above if you can.
+- **Try to avoid** using Social logos at 16px, 17px, or any arbitrary pixel-size that's incompatible with the base 24px grid. For example, don't size the icon font in EMs. 
+
+## Building
+
+In the commandline, type `npm run build`. This will clean up, polish, and generate a build folder that contains the following:
```

Comment:
Definitely not `npm run build`.

For the monorepo it would be `jetpack build js-packages/social-logos`, same as anything else. But having that in the readme for the npm package is kind of bogus.

If we need build instructions in here at all, probably better to put it in a separate md file (CONTRIBUTING.md? BUILDING.md? DEVELOPMENT.md?) that doesn't get included in the mirror.